src/common/transformations/src/transformations/symbolic_transformations/symbolic_optimizations.cpp

```diff
@@ -210,10 +210,23 @@ bool ov::pass::SymbolicOptimizations::run_on_model(const std::shared_ptr<ov::Mod
     // it may break NNCF patterns and lead to unexpected FakeQuantize ops in the model.
     // So we decided to disable these passes in SymbolicOptimizations.
     const auto& pass_config = m_manager->get_pass_config();
+
+    const auto old_pass_config = *pass_config;
```

Comment:
What about the make a copy of config at L:212:
```const auto pass_config = m_manager->get_pass_config();```

then disable some passes on this copy, in case of error the restoration of original pass config should not be required.